### PR TITLE
Express js server

### PR DESCRIPTION
**Pull request by mianjoto**
This is the implementation of the Express.js server which allows us to host our application.

### CODE REVIEW

**Comment by rnguye15** [comment]
Looks good to me!

**Comment by msmit558** [comment]
Looks good!

File: chatroom.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const express = require("express")
+const session = require("express-session")
+const router = express.Router()
+
+router.get("/", (req, res) => {
+    let username = req.session.username;
+    res.render("chatroom", { username })
+})
+
+module.exports = router
```
Comment (Yamez-H) [inline chatroom.js]: does not preform username validation

**Review by Yamez-H — CHANGES_REQUESTED** [review]

File: chatroom.js (hunk 1 above)
Comment (mianjoto) [reply chatroom.js]: Thanks, I'll push the new changes!!

**Review by Yamez-H — APPROVED** [review]

**Pull request merged**